distribution/Dockerfile-ubi

```diff
@@ -23,4 +23,6 @@ COPY ./internal/jobqueue/dbjobqueue/schemas /opt/migrate/schemas
 COPY --from=builder2 /opt/app-root/src/go/bin/tern /opt/migrate/
 
 EXPOSE 8080 8700
-ENTRYPOINT ["python3", "/opt/entrypoint.py", "--remote-worker-api", "--composer-api", "--composer-api-port", "8080"]
+ENV ENTRYPOINT_ADDITIONAL_ARGS "--composer-api-port 8080"
```

Comment:
We can even make `8080` the default value of `--composer-api-port`, I think that's a cleaner way forward as we want to drop TLS from composer.

Reply:
Shouldn't the default port be 80?

Reply:
(it is possible to use one container for both composer and worker API, but we expect these to be run separately, in which case it doesn't matter if their default ports are the same)

Reply:
(it is possible to use one container for both composer and worker API, but we expect these to be run separately, in which case it doesn't matter if their default ports are the same)

Reply:
The entrypoint defined in this Dockerfile starts both services, so it must use separate ports. 
Furthermore, since we allow the user to start both services together, I think it's better to have different default ports so that the user is not forced to define the port numbers.

Reply:
I've implemented it according to @ondrejbudai 's suggestion. The only reason I didn't do it to begin with was not to break previous behavior

Reply:
> Shouldn't the default port be 80?

OpenShift (our primary target) doesn't like running stuff on privileged ports so I think that 8080 is fine.

> (it is possible to use one container for both composer and worker API, but we expect these to be run separately, in which case it doesn't matter if their default ports are the same)

I think that we wanted to consider this in the future but it was never implemented so I wouldn't say that running them separately is expected just yet. Composer in AOC is still running as one container.